README.md

```diff
@@ -3,79 +3,49 @@
-  * __serve__	- Runs a development server
-  * __test__ - Runs tests with web-component-tester
 
 ## Installation
 
-Install via npm:
-
-    $ npm install -g polymer-cli
-
-Then run via `polymer <command>`:
-
-    $ polymer help
-
-## Project Structure
-
-Polymer-CLI is somewhat opinionated about project structure.
-
-There are two type of projects:
-
-* Elements projects
-
-  Element projects contain one or more reusable element definitions, intended to be used from other elements, applications or pages. Element definitions live at the top level of the project so they are easy to import. Elements import their dependencies with relative paths that reference sibling folders to the project folder.
-
-* Application projects
-
-  Application projects are self-contained and intended to be deployed as a standalone website. Application projects contain elements in a `src/` folder and import their dependencies with absolute paths, or relative paths that reference folders inside the project folder.
-
-### Application Styles
-
-Polymer-CLI currently supports two styles of applications:
-
-  * Monolithic applications, which have a single entrypoint (usually index.html) and eagerly import all dependencies.
-
-  * "App shell" applications, which have a very lightweight entrypoint, an app-shell with startup and routing logic, and possibly lazy loaded fragments.
-
-### App-shell Structure
-
-App-shell apps are currently the preferred style for Polymer CLI, and most commands are being optimized to support them. App-shell apps usually have client-side routing (see the [app-route](https://github.com/PolymerElements/app-route) element), and lazy load parts of the UI on demand.
-
-Polymer-CLI supports this style by understand these different types of files:
+```bash
+$ npm install -g polymer-cli
+# or...
+$ yarn add global polymer-cli
+```
 
-  * entrypoint - The first file served by the web server for every valid route (usually index.html). This file should be very small, since it may not cache well and must reference resources with absolute URLs, due to being served from many URLs.
-  * shell - The actual app shell, which includes the top-level logic, routing, and so on.
-  * fragments - lazy loaded parts of the application, typically views and other elements loaded on-demand.
 
 ## Configuration
 
-The project files are specified either as global flags: `--entrypoint`, `--shell` and zero or more `--fragment` flags, or in a `polymer.json` configuration file.
+An application can be configured for all commands via global flags: `--entrypoint`, `--shell`, etc.
 
-### polymer.json
+We recommend saving your configuration to a `polymer.json` configuration file in your project so that they can be read automatically for every command. Other project settings, like build and lint rules, can also be defined here. Read the [polymer.json spec](https://www.polymer-project.org/2.0/docs/tools/polymer-json) for a full list of all supported configurations.
```

Comment:
of all supported configurations -> of all supported fields

`configuration` sounds modal but the linked doc is closer to a listing of fields, what they mean and how to use them.